IBPSA/Airflow/Multizone/Orifice.mo

```diff
@@ -4,10 +4,10 @@ model Orifice "Orifice"
     m=0.5,
     k=CD*A*sqrt(2.0/rho_default));
 
-  parameter Modelica.SIunits.Area A "Area of orifice"
-    annotation (Dialog(group="Orifice characteristics"));
+  parameter Modelica.SIunits.Area A "Area of orifice" annotation (Dialog(group="Orifice characteristics"));
   parameter Real CD=0.65 "Discharge coefficient"
     annotation (Dialog(group="Orifice characteristics"));
+  Modelica.SIunits.Velocity v(nominal=1) "Average velocity";
```

Comment:
Same remark as before: v already exists?

Reply:
The average velocity variable is specific to the type of flow element and was moved to a higher level.  Only EffectiveAirLeakageArea and Orifice use this variable.